### PR TITLE
add make file to make the built es-node versoin contain 

### PR DESCRIPTION
**Pull request by ping-ke**
Related issue: https://github.com/ethstorage/es-node/issues/54

New build:
```
root@goquarkchain-test:~/github.com/ethstorage/es-node# make 
env GO111MODULE=on GOOS= GOARCH= go1.20.10 build -v -ldflags "-X main.GitCommit=eeff04cbffb524bda291abd7006d2b98c43ccbf9 -X main.GitDate=1699671584 -X main.Meta=dev -X 'main.BuildTime=Sat Nov 11 03:03:05 UTC 2023'" -o ./cmd/es-node/es-node ./cmd/es-node/
github.com/ethstorage/go-ethstorage/cmd/es-node
root@goquarkchain-test:~/github.com/ethstorage/es-node# ./cmd/es-node/es-node -v
EthStorage node version v0.1.1-81e083e9-1699873072
Build date: Mon Nov 13 10:59:47 UTC 2023
System version: amd64/linux
Golang version: go1.20.10
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+GITCOMMIT := $(shell git rev-parse HEAD)
+GITDATE := $(shell git show -s --format='%ct')
+VERSION := v0.1.1
+VERSION_META := dev
```
[the remark this answers is not in the document]
Comment (ping-ke) [reply Makefile:4]: the value would like "stable, beta, alpha, dev"

**Comment by qzhodl** [comment]
Can we have a more prettier output:
image
VS
image

File: GUIDE.md [hunk 2]
```diff
@@ -98,7 +98,8 @@ npm install -g snarkjs@0.7.0
 ```
 #### 4. Build es-node
 ```sh
-cd cmd/es-node && go build && cd ../..
+echo VERSION_META would be stable/beta/alpha/dev.
+export VERSION_META=dev && make
```
Comment (qzhodl) [inline GUIDE.md:102]: In most case, we may just make instead of setting the VERSION_META?

**Comment by qzhodl** [comment]
Execute "es-node -v" seems a little slower than excepted

**Review by qzhodl — APPROVED** [review]

**Comment by qizhou** [comment]
GUIDE.md should be also updated accordingly.

**Pull request merged**